### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by dlouseiro**

### CODE REVIEW

**Comment by michael-the1** [comment]
Time for version 1.0? 🤔

**Comment by michael-the1** [comment]
For me it's fine either way, but 1.0 to me implies stability and this has been very stable for a long time now.

**Comment by dlouseiro** [comment]
> For me it's fine either way, but 1.0 to me implies stability and this has been very stable for a long time now.

Indeed! Let's go for 1.0.

We're only missing one approval on [this one](https://github.com/PicnicSupermarket/diepvries/pull/55)!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (matthieucan) [inline CHANGELOG.md:7]: Let's add a new (empty) section with "Unreleased", for future improvements

Comment (dlouseiro) [reply CHANGELOG.md:7]: Sure!

Comment (dlouseiro) [reply CHANGELOG.md:7]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [1.0.0]
```
Comment (matthieucan) [inline CHANGELOG.md:9]: Should we add a mention of the change done in #55?

**Review by matthieucan — APPROVED** [review]
Little question, but approving. Great achievement for to release `1.0.0`! 🚀

File: CHANGELOG.md (hunk 2 above)
Comment (dlouseiro) [reply CHANGELOG.md:9]: Sure, will do! WIll also mention this [one](https://github.com/PicnicSupermarket/diepvries/pull/53)

**Review by matthieucan — APPROVED** [review]

**Pull request merged**